_posts/2011-04-13-odata-service-validation-tool.html

```diff
@@ -11,7 +11,7 @@
 author: OData.org Archive
 ---
 <p>OData is released under the Microsoft Open Specification promise. This allows anyone to create OData services that implement the specification and to freely interoperate with OData implementations.</p>
-<p>Currently there are several OData service <a href="https://www.odata.org/ecosystem">producers</a> and <a href="assets/odata-sdk">server libraries</a> including .Net Framework, Java, Rails and several <a href="https://www.odata.org/developers/odata-sdk">client libraries</a> across a range of platforms such as Objective-C, Javascript, PHP, and Java. The fact that an OData service can be consumed by a wide range of applications and libraries makes interoperability a key requirement.</p>
+<p>Currently there are several OData service <a href="/ecosystem">producers</a> and <a href="assets/odata-sdk">server libraries</a> including .Net Framework, Java, Rails and several <a href="/developers/odata-sdk">client libraries</a> across a range of platforms such as Objective-C, Javascript, PHP, and Java. The fact that an OData service can be consumed by a wide range of applications and libraries makes interoperability a key requirement.</p>
 <p>Today we are announcing the <a href="http://validator.odata.org/">OData Service Validation Tool</a> to address that requirement. The goal with this tool is to enable OData service authors ensure that their service interoperates well with any OData client. Consumers of the OData protocol can also benefit from this tool by testing OData service implementations that they are building an experience for to pinpoint potential issues.</p>
```

Comment:
I've tried to visit `http://validator.odata.org` but it appears to be a dead link. I don't know if you know much about the service or where it's hosted. Maybe that's outside the scope of this PR. But I wondered how such URLs (with subdomains) should be handled.

Reply:
@habbes This is a dead link. I came across it and opted to leave it as it is. There was no sense in converting it into relative URL if that too would result into a 404. I also didn't have a valid alternative I could change it to.